python/interpret_text/common/preprocessor/glove_preprocessor.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import pandas as pd
+from typing import Any, Iterable, Tuple, List
+from interpret_text.common.base_explainer import BaseTextPreprocessor
+
+
+class GlovePreprocessor(BaseTextPreprocessor):
+
+    def __init__(self, count_threshold: int, token_cutoff: int):
+        self.count_threshold = count_threshold
+        self.token_cutoff = token_cutoff
+
+    def build_vocab(self, text) -> Any:
+        words_to_indexes = {"<PAD>": 0, "<UNK>": 1}
+        counts = {}
+        for sentence in text:
+            for word in sentence.split():
+                if word not in words_to_indexes:
```

Comment:
code looks similar to BertPreprocessor, consider refactoring common logic to helper functions to reduce code duplication